packages/form-builder/src/common/getWindowData.ts

```diff
@@ -6,10 +6,21 @@ declare global {
             formDesigns: FormDesign[];
             formPage: FormPageSettings;
             currency: string;
-        };
+        },
+        formBuilderData?: {
+            gateways: Array<any>;
```

Comment:
let's make a typescript `Gateway` type/interface with id, label, supportsSubscriptions.

Then here we can do `gateways: Gateway[]`

The other areas you are using this should then be aware of the properties of the gateway object.

Reply:
Resolved, see https://github.com/impress-org/givewp-next-gen/pull/140/commits/936806a7864c0237645d3d4f813d8a76413cda93.